src/rust/ensogl/lib/core/src/display/shape/primitive/def/primitive.rs

```diff
@@ -203,6 +203,28 @@ define_sdf_shapes! {
     }
 
 
+    // === RoundedLineSegment ===
+
+    /// A line segment from the origin to `target` with rounded endpoints.
+    RoundedLineSegment (target:Vector2<Pixels>, width:Pixels) {
```

Comment:
Why "LineSegment" and not just "Line" ?

Reply:
Also, I asked you to provide info where these equation come from. Please next to each such function link to sources.

Reply:
The word "line" commonly refers to lines that run to infinity, while the line segment has two ends.

Reply:
Hmm, in graphics we nomaly use line for line segment, but as in SDF there is something like infinite line, then yeah, lets leave this name then :)